docker-compose.yml

```diff
@@ -1,19 +1,46 @@
 version: '3.7'
 
+x-defaults: &x-defaults
+  environment:
+    BINDING: 0.0.0.0
+    PORT: 3000
+    POSTGRES_HOST: "${POSTGRES_HOST:-db}"
+    POSTGRES_PORT: "${POSTGRES_PORT:-5432}"
+    SIGNAL_REST_CLI_ENDPOINT: "http://signal:8080"
+    SIGNAL_REST_CLI_ATTACHMENT_PATH: "signal-cli-config/attachments/"
+
 services:
   app:
+    <<: *x-defaults
+    command: bundle exec rails server
+    depends_on: [ db, queue ]
+
+  queue:
```

Comment:
We should rename this to `delayed_job`, `worker`, `queue_worker` or something like that, just for clarity.

Reply:
Yes please. And I'm already weary of the future troubles we will get from the extra service.